resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "lonesome_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["lonesome"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c slinks forward with an air of unsettling detachment. Rarely does a sound escape {PRONOUN/r_c/poss} lips, as this feline only speaks to hand m_c a life for [virtue], then fades into the shadows once more. {PRONOUN/r_c/poss/CAP} haunting presence lingers, leaving behind a trail of curiosity and unspoken whispers.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunningness", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        },
+        {
+          "text": "Though lacking words, r_c's presence emanates an unspoken offer of a life for m_c, embedded with the virtue of [virtue]. The departure leaves a trail of mystery and reverential awe in r_c's aftermath.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunningness", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        },
+        {
+          "text": "With steady footing, r_c approaches m_c, {PRONOUN/r_c/poss} velvet paws treading noiselessly upon the moss-covered ground. Brooding demeanor, accompanied by an ominous silence, alludes to a dark past that remains locked within {PRONOUN/r_c/poss} enigmatic spirit. {PRONOUN/r_c/subject/CAP} {VERB/r_c/give/gives} a life for [virtue], then {VERB/r_c/leave/leaves} without a sound escaping {PRONOUN/r_c/poss} tongue.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunningness", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "loving_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["loving"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c approaches and entwines {PRONOUN/r_c/poss} tail with m_c's. {PRONOUN/r_c/subject/CAP} {VERB/r_c/promise/promises} m_c that {PRONOUN/m_c/poss} destiny awaits, if only {PRONOUN/m_c/subject} will be ruthless enough to seize it. r_c gives a life for [virtue].",
```

```suggestion
          "text": "r_c approaches and entwines {PRONOUN/r_c/poss} tail with {PRONOUN/m_c/inposs}. {PRONOUN/r_c/subject/CAP} {VERB/r_c/promise/promises} m_c that {PRONOUN/m_c/poss} destiny awaits, if only {PRONOUN/m_c/subject} will be ruthless enough to seize it. r_c gives a life for [virtue].",
```

Correction and **ooh**, it'd be nice if we could tie in the virtue to the promise. Maybe something like "r_c feels sure that this life for [virtue] will be the first pawprint on the path."